### PR TITLE
add cocospod surpport for ncnn

### PR DESCRIPTION
**Pull request by DaChengTechnology**
add release pod ci and Mac&&iOS podspec

### CODE REVIEW

**Comment by DaChengTechnology** [comment]
fixed Mac pod

**Comment by DaChengTechnology** [comment]
if you marged this pull request,then give me a url for iOS_openmp_bitcode after run the realse_pod.yml. then I`ll update dpcument and examples @nihui

**Comment by DaChengTechnology** [comment]
test the cocoapods publish is passed

**Comment by DaChengTechnology** [comment]
pod search NCNN

File: NCNN_IOS.podspec [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+  spec.homepage     = "https://github.com/Tencent/ncnn"
+
+  spec.license      = "BSD 3-Clause"
+  spec.license      = { :type => "BSD 3-Clause", :file => "https://github.com/Tencent/ncnn/LICENSE.txt" }
+
+  spec.author             = { "DCTech" => "412200533@qq.com" }
+
+  spec.platform     = :ios
+  spec.platform     = :ios, "9.0"
```
Comment (nihui) [inline NCNN_IOS.podspec:19]: duplicated license and platform

File: NCNN_MAC.podspec [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+  spec.homepage     = "https://github.com/Tencent/ncnn"
+
+  spec.license      = "BSD 3-Clause"
+  spec.license      = { :type => "BSD 3-Clause", :file => "https://github.com/Tencent/ncnn/LICENSE.txt" }
+
+  spec.author             = { "DCTech" => "412200533@qq.com" }
+
+  spec.platform     = :osx
+  spec.platform     = :osx, "10.12"
```
Comment (nihui) [inline NCNN_MAC.podspec:19]: duplicated license and platform

**Review by nihui — CHANGES_REQUESTED** [review]

**Comment by DaChengTechnology** [comment]
it is fixed

**Pull request closed without merging**